packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -36,6 +36,26 @@ export const ActivityIndicator = compose<ActivityIndicatorType>({
     // hiding opacity makes the screen reader on iOS and Android skip over it
     const hideOpacity = animating == false && hidesWhenStopped == true ? 0 : 1;
 
+    const useNativeDriver: boolean = (() => {
+      // Allow the user to override the default setting if needed
+      if (props.useNativeDriver !== undefined) {
+        return props.useNativeDriver;
+      }
+      /**
+       * https://github.com/facebook/react-native/pull/29585
+       * In order for native driven animations to loop, React Native needs this fix,
+       * which is only available in React Native 0.66+, and React Native macOS 0.62+
+       */
+      const nativeVersion = Platform.constants.reactNativeVersion;
+      if (Platform.OS === 'macos' && nativeVersion.minor >= 62) {
+        return true;
+      } else if (nativeVersion.minor >= 66) {
+        return true;
+      } else {
+        return false;
+      }
```

Comment:
This isn't a perfect check of whether we are running with react-native-macos or not: An iOS or Android app could be running with react-native-macos 0.63, and this would return false where it should return true. I couldn't think of a better way to do it though.